client/src/pages/login/index.js

```diff
@@ -1,33 +1,88 @@
 import React, { Component } from "react";
+import SweetAlert from "react-bootstrap-sweetalert";
 import { Redirect, withRouter } from "react-router-dom";
 
 import { Input, Button } from "../../components";
 import "./login.css";
 
 class Login extends Component {
   state = {
-    isLogin: false,
+    isLoggedIn: false,
+    alert: null,
     loginInfo: {}
   };
 
+  invalidPasswordOrUsernameAlert() {
+    const getAlert = () => (
+      <SweetAlert
+        warning
+        title="Wrong!"
+        confirmBtnBsStyle="danger"
+        s
+        cancelBtnBsStyle="danger"
+        onConfirm={() => this.hideAlert()}
+      >
+        Invalied Password or Username !
+      </SweetAlert>
+    );
+
+    this.setState({
+      alert: getAlert()
```

Comment:
(I'm copying this comment over too, from #26)

Instead of storing the alert component in state, you might want to store a boolean (showLoginError) and put the SweetAlert component here in render() if state.showLoginError is true.
The react docs recommend that you don't store components in state: http://web.archive.org/web/20150419023006/http://facebook.github.io/react/docs/interactivity-and-dynamic-uis.html#what-shouldnt-go-in-state